packages/babel-plugin-proposal-private-methods/test/fixtures/private-method/exfiltrated/exec.js

```diff
@@ -0,0 +1,13 @@
+let exfiltrated;
+class Foo {
+    #privateMethod() {}
+
+    constructor() {
+        if (exfiltrated === undefined) {
+            exfiltrated = this.#privateMethod;
+        }
+        expect(exfiltrated).toStrictEqual(this.#privateMethod);
+    }
+}
+
+new Foo();
```

Comment:
Can you add another `new Foo()`? So it also checks that the private method is the same accross different instances.

Reply:
Added to both strict and loose.